CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to `calendar-links` will be documented in this file
 
+## 1.1.1 - 2018-10-08
```

Comment:
Please don't modify the changelog in this PR, I'll take care of this when it gets released :)

Reply:
![image](https://user-images.githubusercontent.com/5278175/47232597-c23d6400-d3d8-11e8-8f27-ce505a74fdb8.png)
that's not my commit. But maybe I did something wrong on rebasing from upstream